tests/test_logger.py

```diff
@@ -0,0 +1,86 @@
+"""
+Tests for the ralph.logger module
+"""
+
+from pathlib import Path
+
+from click.testing import CliRunner
+
+import ralph.logger
+from ralph.cli import cli
+from ralph.exceptions import ConfigurationException
+
+
+# pylint: disable=invalid-name, unused-argument
+def test_logger_exists(fs, monkeypatch):
+    """Test the logging system when a correct configuration is provided"""
+
+    mock_default_config = {
+        "version": 1,
+        "propagate": True,
+        "formatters": {
+            "ralph_format": {
+                "format": "%(asctime)-23s %(levelname)-8s %(name)-8s %(message)s"
+            },
+        },
+        "handlers": {
+            "ralph_stderr": {
+                "class": "logging.StreamHandler",
+                "level": "INFO",
+                "stream": "ext://sys.stderr",
+                "formatter": "ralph_format",
+            },
+        },
+        "loggers": {
+            "ralph": {
+                "handlers": ["ralph_stderr"],
+                "level": "INFO",
+            },
+        },
+    }
+
+    Path("dev").mkdir()
+    Path("app/.ralph").mkdir(parents=True)
```

Comment:
Is this required? If yes, use the `fs` fixture helpers instead: http://jmcgeheeiv.github.io/pyfakefs/release/usage.html#using-convenience-methods